### PR TITLE
suzanne's edits: first pass

### PR DESCRIPTION
**Pull request by sascala**
Still to do: change uses of "we" to "you."

### CODE REVIEW

**Comment by sascala** [comment]
Note: some of the external links seem to have extra characters that break them. I haven't yet been able to figure out why.

File: docs/tutorial.md [hunk 1]
```diff
@@ -2,44 +2,50 @@
...
 scheduler is run by Marathon, which supervises the scheduler,
-restarting it upon failure.  The executors are started and supervised
-by the scheduler.  If specified in the `ServiceSpecification`, the
+restarting it upon failure.  The scheduler, in turn, starts and
+supervises the executors.  If specified in the `ServiceSpecification`,
+the
 executors will contain a **persistent volume**, which is a folder or
-mount point on the agent that persists even after a task fails.
+mount point on the agent that persists even after a task fails. This
+allows a stateful service to recover with its data intact after node
```
Comment (mgummelt) [inline docs/tutorial.md]: "after task failure or temporary node failure"

If it's a permanent failure, we can't recover the volume.

File: docs/tutorial.md [hunk 2]
```diff
@@ -71,24 +77,26 @@ repositories {
     maven {
         url "http://downloads.mesosphere.com/maven/"
     }
-}
-
+    }    }    }}
+    }    }    }}    }    }}    }}
```
Comment (mgummelt) [inline docs/tutorial.md]: ???

File: docs/tutorial.md [hunk 3]
```diff
@@ -97,18 +105,27 @@ ordering to define tiers of services.  In this example, we want all
 
 ```java
 private static final String TASK_METADATA_NAME = "meta-data";
-private static final int TASK_METADATA_COUNT = Integer.valueOf(System.getenv("METADATA_COUNT"));
-private static final double TASK_METADATA_CPU = Double.valueOf(System.getenv("METADATA_CPU"));
-private static final double TASK_METADATA_MEM_MB = Double.valueOf(System.getenv("METADATA_MEM"));
-private static final double TASK_METADATA_DISK_MB = Double.valueOf(System.getenv("METADATA_DISK"));
-private static final String TASK_METADATA_URI = System.getenv("METADATA_URI");
+private static final int TASK_METADATA_COUNT =
```
Comment (mgummelt) [inline docs/tutorial.md]: Why split up these lines?  They aren't split in the code, and doing so would be non-standard Java.

File: docs/tutorial.md [hunk 4]
```diff
@@ -155,7 +177,8 @@ server:
 ```java
 private static Protos.CommandInfo getCommand(String cmd, String uri) {
     return Protos.CommandInfo.newBuilder()
-              .addUris(Protos.CommandInfo.URI.newBuilder().setValue(uri))
+              .addUris(Protos.CommandInfo.URI.newBuilder().setValue(
```
Comment (mgummelt) [inline docs/tutorial.md]: Lots of unnecessary line breaks.

File: docs/tutorial.md [hunk 5]
```diff
@@ -608,118 +687,144 @@ curl "data-store.marathon.mesos:<port>/v1/plan/status"
...
-task is backup/restore.  The service must allocate resources on the
-executor to perform this task, restart it upon failure, and coordinate
-it across all executors.
+against: the scheduler and the service. The SDK sets up fault
+tolerance for your scheduler automatically.
+
+The SDK abstracts away task reconciliation with the Mesos master in
+the case of task failure. In the case of service failure, the SDK
+configures your scheduler to restart the task on the same node or, if
+the node has failed, restarts the task on a different node.
```
Comment (mgummelt) [inline docs/tutorial.md]: We should emphasize that the persistent volume is reused on a restart, and a new one is created when the node fails.

**Comment by mgummelt** [comment]
Thanks, lots of improvements here.

**Comment by sascala** [comment]
OK, @mgummelt, I've incorporated your comments and fixed the code blocks.

**Comment by mgummelt** [comment]
> Still to do: change uses of "we" to "you."

Lol, I went through and did the opposite before I handed it off to you.  Your call.

File: docs/tutorial.md [hunk 6]
```diff
@@ -608,118 +667,151 @@ curl "data-store.marathon.mesos:<port>/v1/plan/status"
...
+tolerance for your scheduler automatically.
+
+The SDK abstracts away task reconciliation with the Mesos master in
+the case of task failure. In the case of service failure, the SDK
+configures your scheduler to restart the task on the same node or, if
+the node has failed, restarts the task on a different node. The
+scheduler's persistent volume is reused on restart. A new persistent
+volume is created when a node fails.
+
+
```
Comment (mgummelt) [inline docs/tutorial.md]: extra newline

File: docs/tutorial.md [hunk 7]
```diff
@@ -608,118 +667,151 @@ curl "data-store.marathon.mesos:<port>/v1/plan/status"
...
+We should emphasize that the persistent volume is reused on a restart,
```
Comment (mgummelt) [inline docs/tutorial.md]: I didn't mean you had to include my comment verbatim :)

**Comment by mgummelt** [comment]
LGTM

feel free to merge

**Pull request merged**